TombLib/TombLib/Utils/ImageC.cs

```diff
@@ -624,11 +624,10 @@ private static unsafe uint ColorToUint(ColorC color)
             return *(uint*)byteArray;
         }
 
-        public unsafe void ReplaceColor(ColorC from, ColorC to)
+        public unsafe void ReplaceColor(in ColorC from,in ColorC to)
         {
             uint fromUint = ColorToUint(from);
             uint toUint = ColorToUint(to);
-
```

Comment:
Like before, please always keep a space before any block declaration and after a block chain is finished (`if, else if, else`).